.github/workflows/goth.yml

```diff
@@ -18,12 +31,15 @@ jobs:
   integration-test:
     name: Integration Tests
     runs-on: goth
-    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    needs: wait-for-build
     defaults:
       run:
         working-directory: './goth_tests'
 
     steps:
+      - run: echo '${{ toJson(github) }}'
```

Comment:
Pls make sure we do not reveal `github.token` by echoing whole `github` context.
See warning here:
https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context

Reply:
for sure! is already removed

this was just for debugging.. will keep it in mind next time